app/views/users/show.html.erb

```diff
@@ -0,0 +1,34 @@
+
+<h1><%=@user.user_name%></h1>
+
+<!-- <%= link_to "Edit Profile", edit_user_registration_path %> -->
+<% if @user.avatar %>
+  <img src="<%= @user.avatar %>"/>
+<% else %>
+  <%= image_tag "/assests/default_pic.jpg"%>
+<%end%>
+
+<h3>Favorite Show: </h3>
+<% if @user.favorite_show %>
+  <%= @user.favorite_show %>
+<%else%>
+<h4>Not yet decided</h4>
+<%end%>
+
+
+<h3>Show Reviews:</h3>
+<% if @reviews.empty? %>
+  <h4>So much empty..Start Reviewing!</h4>
+<% else %>
+<% @reviews.each do |review| %>
+<ul>
+  <li>
+    Show: <%=review.program.title%> <br>
+    Review: <%=review.headline%> <br>
+    Why?: <%=review.body %> <br>
+      Rating: <%=review.rating %> <br>
+      Vote Count: <%=review.vote_count%>
+  </li>
+</ul>
+<%end%>
+<%end%>
```

Comment:
Please fix your indentation at the end and your spacing after a bunch of your erb tags: `<% end %>`